### PR TITLE
Changes based on style and lint checks. (final_code_quality_2)

### PR DESCRIPTION
**Pull request by coffeedogs**
Changes based on style and lint checks. (final_code_quality_2)

### CODE REVIEW

File: src/bitmessageqt/settingsmixin.py [hunk 1]
```diff
@@ -1,79 +1,105 @@
...
             target.restoreState(state.toByteArray() if hasattr(state, 'toByteArray') else state)
-        except Exception as e:
+        except Exception:
             pass
 
-            
+
 class SMainWindow(QtGui.QMainWindow, SettingsMixin):
+    """"""
     def loadSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Load main window settings.

File: src/bitmessageqt/settingsmixin.py [hunk 2]
```diff
@@ -1,79 +1,105 @@
...
+        """"""
         self.readGeometry(self)
         self.readState(self)
-        
+
     def saveSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Save main window settings

File: src/bitmessageqt/settingsmixin.py [hunk 3]
```diff
@@ -1,79 +1,105 @@
...
+        """"""
         self.writeState(self)
         self.writeGeometry(self)
 
 
 class STableWidget(QtGui.QTableWidget, SettingsMixin):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Table widget with Settings functionality

File: src/bitmessageqt/settingsmixin.py [hunk 4]
```diff
@@ -1,79 +1,105 @@
...
+    """"""
+    # pylint: disable=too-many-ancestors
     def loadSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Load table settings.

File: src/bitmessageqt/settingsmixin.py [hunk 5]
```diff
@@ -1,79 +1,105 @@
...
+        """"""
         self.readState(self.horizontalHeader())
 
     def saveSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Save table settings.

File: src/bitmessageqt/settingsmixin.py [hunk 6]
```diff
@@ -1,79 +1,105 @@
...
+        """"""
         self.writeState(self.horizontalHeader())
 
-        
+
 class SSplitter(QtGui.QSplitter, SettingsMixin):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Splitter with Settings functionality.

File: src/bitmessageqt/settingsmixin.py [hunk 7]
```diff
@@ -1,79 +1,105 @@
...
+    """"""
     def loadSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Load splitter settings

File: src/bitmessageqt/settingsmixin.py [hunk 8]
```diff
@@ -1,79 +1,105 @@
...
+        """"""
         self.readState(self)
 
     def saveSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Save splitter settings.

File: src/bitmessageqt/settingsmixin.py [hunk 9]
```diff
@@ -1,79 +1,105 @@
...
+        """"""
         self.writeState(self)
-        
+
 
 class STreeWidget(QtGui.QTreeWidget, SettingsMixin):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Tree widget with settings functionality.

File: src/bitmessageqt/settingsmixin.py [hunk 10]
```diff
@@ -1,79 +1,105 @@
...
+    """"""
+    # pylint: disable=too-many-ancestors
     def loadSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Load tree settings.

File: src/bitmessageqt/settingsmixin.py [hunk 11]
```diff
@@ -1,79 +1,105 @@
...
-        #recurse children
-        #self.readState(self)
+        """"""
+        # recurse children
+        # self.readState(self)
         pass
 
     def saveSettings(self):
```
Comment (PeterSurda) [inline src/bitmessageqt/settingsmixin.py]: Save tree settings

File: src/network/socks5.py [hunk 12]
```diff
@@ -1,36 +1,48 @@
...
+# pylint: disable=attribute-defined-outside-init
+
 import socket
 import struct
 
-from proxy import Proxy, ProxyError, GeneralProxyError
+from proxy import GeneralProxyError, Proxy, ProxyError
+
 
 class Socks5AuthError(ProxyError):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Thrown when the socks5 protocol encounters an authentication error

File: src/network/socks5.py [hunk 13]
```diff
@@ -1,36 +1,48 @@
...
-        "All offered authentication methods were rejected",
-        "Unknown username or invalid password",
-        "Unknown error")
+                  "Authentication is required",
+                  "All offered authentication methods were rejected",
+                  "Unknown username or invalid password",
+                  "Unknown error")
 
 
 class Socks5Error(ProxyError):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Thrown when socks5 protocol encounters an error

File: src/network/socks5.py [hunk 14]
```diff
@@ -1,36 +1,48 @@
...
 
 
 class Socks5(Proxy):
+    """"""
     def __init__(self, address=None):
         Proxy.__init__(self, address)
         self.ipaddr = None
         self.destport = address[1]
 
     def state_init(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Protocol initialisation (before connection is established)

File: src/network/socks5.py [hunk 15]
```diff
@@ -39,6 +51,7 @@ def state_init(self):
         return True
 
     def state_auth_1(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Perform authentication if peer is requesting it.

File: src/network/socks5.py [hunk 16]
```diff
@@ -62,6 +75,7 @@ def state_auth_1(self):
         return True
 
     def state_auth_needed(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle response to authentication attempt

File: src/network/socks5.py [hunk 17]
```diff
@@ -74,14 +88,15 @@ def state_auth_needed(self):
         return True
 
     def state_pre_connect(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle feedback from socks5 while it is connecting on our behalf.

File: src/network/socks5.py [hunk 18]
```diff
@@ -96,21 +111,25 @@ def state_pre_connect(self):
         return True
 
     def state_proxy_addr_1(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle IPv4 address returned for peer

File: src/network/socks5.py [hunk 19]
```diff
@@ -96,21 +111,25 @@ def state_pre_connect(self):
...
+        """"""
         self.boundaddr = self.read_buf[0:4]
         self.set_state("proxy_port", length=4, expectBytes=2)
         return True
 
     def state_proxy_addr_2_1(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle other addresses than IPv4 returned for peer (e.g. IPv6, onion, ...). This is part 1 which retrieves the length of the data.

File: src/network/socks5.py [hunk 20]
```diff
@@ -96,21 +111,25 @@ def state_pre_connect(self):
...
+        """"""
         self.address_length = ord(self.read_buf[0:1])
         self.set_state("proxy_addr_2_2", length=1, expectBytes=self.address_length)
         return True
 
     def state_proxy_addr_2_2(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle other addresses than IPv4 returned for peer (e.g. IPv6, onion, ...). This is part 2 which retrieves the data.

File: src/network/socks5.py [hunk 21]
```diff
@@ -96,21 +111,25 @@ def state_pre_connect(self):
...
+        """"""
         self.boundaddr = self.read_buf[0:self.address_length]
         self.set_state("proxy_port", length=self.address_length, expectBytes=2)
         return True
 
     def state_proxy_port(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle peer's port being returned.

File: src/network/socks5.py [hunk 22]
```diff
@@ -121,14 +140,17 @@ def state_proxy_port(self):
         return True
 
     def proxy_sock_name(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Handle DNS return value if using SOCKS5 for resolving instead of connecting.

File: src/network/socks5.py [hunk 23]
```diff
@@ -121,14 +140,17 @@ def state_proxy_port(self):
...
+        """"""
         return socket.inet_ntoa(self.__proxysockname[0])
 
 
 class Socks5Connection(Socks5):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Child socks5 class used for making outbound connections.

File: src/network/socks5.py [hunk 24]
```diff
@@ -121,14 +140,17 @@ def state_proxy_port(self):
...
+    """"""
```
Comment (PeterSurda) [inline src/network/socks5.py]: Init

File: src/network/socks5.py [hunk 25]
```diff
@@ -121,14 +140,17 @@ def state_proxy_port(self):
...
     def __init__(self, address):
         Socks5.__init__(self, address=address)
 
     def state_auth_done(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Request connection to be made

File: src/network/socks5.py [hunk 26]
```diff
@@ -151,6 +175,7 @@ def state_auth_done(self):
         return True
 
     def state_pre_connect(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Tell socks5 to initiate a connection

File: src/network/socks5.py [hunk 27]
```diff
@@ -159,12 +184,14 @@ def state_pre_connect(self):
 
 
 class Socks5Resolver(Socks5):
```
Comment (PeterSurda) [inline src/network/socks5.py]: DNS resolver class using socks5

File: src/network/socks5.py [hunk 28]
```diff
@@ -159,12 +184,14 @@ def state_pre_connect(self):
...
+    """"""
     def __init__(self, host):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Init

File: src/network/socks5.py [hunk 29]
```diff
@@ -159,12 +184,14 @@ def state_pre_connect(self):
...
         self.host = host
         self.port = 8444
         Socks5.__init__(self, address=(self.host, self.port))
 
     def state_auth_done(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Perform reslving

File: src/network/socks5.py [hunk 30]
```diff
@@ -173,4 +200,5 @@ def state_auth_done(self):
         return True
 
     def resolved(self):
```
Comment (PeterSurda) [inline src/network/socks5.py]: Resolving is done, process the return value. To use this within PyBitmessage, a callback needs to be implemented which hasn't been done yet.

File: src/network/socks5.py [hunk 31]
```diff
@@ -1,36 +1,48 @@
+"""
+src/network/msocks5.py
```
Comment (PeterSurda) [inline src/network/socks5.py]: typo

File: src/network/socks5.py [hunk 32]
```diff
@@ -121,14 +146,17 @@ def state_proxy_port(self):
         return True
 
     def proxy_sock_name(self):
+        """Handle eturn value when using SOCKS5 for DNS resolving instead of connecting."""
```
Comment (PeterSurda) [inline src/network/socks5.py]: typo

File: src/network/socks5.py (hunk 24 above)
Comment (coffeedogs) [reply src/network/socks5.py]: Went for """Child socks5 class used for making outbound connections.""", assuming you were thinking about commenting the `__init__` method with your comment

**Pull request merged**